src/tests/functional/visit_tests.js

```diff
@@ -62,11 +65,81 @@ test("visiting a location served with a non-HTML content type", async ({ page })
   const contentType = await contentTypeOfURL(url)
   assert.equal(contentType, "image/svg+xml")
 
+  assert.deepEqual(
+    requestedUrls, [
+    ['document', 'http://localhost:9000/src/tests/fixtures/svg.svg']
+  ]
+  )
```

Comment:
Just some indentation tweaks and switching to double quotes:

```suggestion
  assert.deepEqual(requestedUrls, [
    ["document", "http://localhost:9000/src/tests/fixtures/svg.svg"]
  ])
```